### PR TITLE
feat: Added serving size to print view

### LINKED ISSUE #3788: [BUG] - Missing serving size when printing recipe
### First Check

- [X] This is not a feature request.
- [X] I added a very descriptive title to this issue (title field is above this).
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [X] This issue can be replicated on the demo site (https://demo.mealie.io/).

### What is the issue you are experiencing?

When a recipe is printed (to paper or PDF -- doesn't matter), the serving size is missing.

### Steps to Reproduce

1) Open a recipe
2) Click the blue circle with the three dots that is on the bottom right of the recipe image.
3) Choose "print"
4) Both the preview and the print result do not show the serving size.

The "print preferences" view also does not show the serving size

### Please provide relevant logs

No relevant logs

### Mealie Version

Docker image ghcr.io/mealie-recipes/mealie, v1.9.0

### Deployment

Docker (Linux)

### Additional Deployment Details

_No response_

### PR DESCRIPTION
**Pull request by thomaspijper**
## What type of PR is this?

- feature

## What this PR does / why we need it:

This modification to `RecipePrintView.vue` adds the serving size to the print view.

image

## Which issue(s) this PR fixes:

This fixes #3788.

## Special notes for your reviewer:

The location of the serving size in the recipe (currently directly below the "Ingredients" header) is of course a matter of opinion. This was what made most sense to me.

## Testing

Change was tested in Docker container built from the branch.

### CODE REVIEW

File: frontend/components/Domain/Recipe/RecipePrintView.vue [hunk 1]
```diff
@@ -30,6 +30,9 @@
     <!-- Ingredients -->
     <section>
       <v-card-title class="headline pl-0"> {{ $t("recipe.ingredients") }} </v-card-title>
+      <div class="font-italic px-0 py-0">
+              <SafeMarkdown :source="recipe.recipeYield" />
+      </div>
```
Comment (boc-the-git) [inline frontend/components/Domain/Recipe/RecipePrintView.vue]:
```suggestion
      <div class="font-italic px-0 py-0">
        <SafeMarkdown :source="recipe.recipeYield" />
      </div>
```

**Review by boc-the-git — CHANGES_REQUESTED** [review]
Just a formatting thing. Otherwise this looks good to me. Thanks for going ahead and raising the PR yourself, appreciate it!

I also agree, this is a potentially contentious thing in terms of where the info goes on the print out, but I think you've chosen the right place 👍

**Comment by boc-the-git** [comment]
Oh, I forgot to add, please let me know the link for that recipe - you've piqued my curiosity!

**Review by boc-the-git — APPROVED** [review]

**Comment by thomaspijper** [comment]
Thanks for reviewing. I accepted the requested change.

Glad I could be of help.

The recipe is: https://www.onceuponachef.com/recipes/new-york-style-cheesecake.html

**Pull request merged**